modules/codelite/tests/test_codelite_config.lua

```diff
@@ -126,12 +127,16 @@
 
 	function suite.OnProjectCfg_ResInclude()
 		files { "x.rc" }
-		resincludedirs { "dir/" }
+		includedirs { "dir/" }
+		sysincludedirs { "sysdir/" }
```

Comment:
This test was for `resincludedirs`, create a new test.

Reply:
I saw it as all relative to resource include directives.
split anyway.